Laravel.gitignore

```diff
@@ -18,3 +18,13 @@ Homestead.json
 
 # Rocketeer PHP task runner and deployment package. https://github.com/rocketeers/rocketeer
 .rocketeer/
+yarn-error.log
+
+#The .idea folder (hidden on OS X) in the solution root contains IntelliJ’s project specific settings files
+/.idea
```

Comment:
Why including this line if it's not related with the Laravel Framework?

Reply:
This is covered by [`Global\JetBrains.gitignore`](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore).